tool_commands/commands/Step 1 - Check data/schematisation_checker.py

```diff
@@ -83,14 +83,25 @@ def run_it(self, threedi_db):
 
         total_checks = len(model_checker.config.checks)
         with progress_bar(self.iface, max_value=total_checks) as pb, open(
-            output_file_path, "w"
+            output_file_path, "w", newline=""
         ) as output_file:
+            writer = csv.writer(output_file)
+            writer.writerow(
+                ["check_type", "table", "column", "id", "value", "description"]
```

Comment:
```suggestion
                ["type of check", "table", "column", "id", "value", "description"]
```

Is het niet lekkerder om column en id om te draaien? Je kijkt vaak eerst naar de regel, toch?
En dan zitten "column" en "value" ook lekkerder bij elkaar.

Reply:
Ja, ik heb ook 'type of check' achteraan geplaatst want ik verwacht dat die voor gebruikers ook minder relevant is.